js_modules/dagster-ui/packages/ui-core/src/assets/AssetPageHeader.oss.tsx

```diff
@@ -71,8 +74,18 @@ export const AssetPageHeader = ({
       return href;
     }, '/assets');
 
-    return list;
-  }, [assetKey.path, headerBreadcrumbs, location.search]);
+    // Prepend the basePath on all items. We don't have control over the breadcrumb
+    // overflow rendering, and Blueprint renders the overflow items with no awareness
+    // of the basePath. This allows us to render appropriate href values for the
+    // overflow items, and we can then remove the basePath for individual rendered
+    // breadcrumbs, which we are able to control.
+    return list.map((item) => {
+      return {
+        ...item,
+        href: `${basePath}${item.href}`,
```

Comment:
I think maybe `createHref` also adds the basePath?

Reply:
Yep, thats how I fixed this issue in https://github.com/dagster-io/dagster/pull/16954/files